doc/source/io.rst

```diff
@@ -4633,6 +4636,41 @@ Read only certain columns of a parquet file.
    os.remove('example_pa.parquet')
    os.remove('example_fp.parquet')
 
+
+Handling Indexes
+''''''''''''''''
+
+Dumping a ``DataFrame`` to parquet may include the implicit index as one or more
```

Comment:
can u add a version changed tag

Reply:
not sure we use the word Dumping anywhere
use serialize

Reply:
Done